### PR TITLE
[v6r13] New class to handle the revoked certificates 

### PR DESCRIPTION
**Pull request by zmathe**
It is used by the new portal...

### CODE REVIEW

File: Core/Security/X509CRL.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        os.write( fd, pemData )
+        os.close( fd )
+      else:
+        fd = file( filename, "w" )
+        fd.write( pemData )
+        fd.close()
+    except Exception, e:
+      return S_ERROR( "Cannot write to file %s :%s" % ( filename, str( e ) ) )
+    try:
+      os.chmod( filename, stat.S_IRUSR | stat.S_IWUSR )
```
Comment (fstagni) [inline Core/Security/X509CRL.py]: I don't see "import stat" anywhere...

Run pylint on this file please.

Comment (zmathe) [inline Core/Security/X509CRL.py]: Done

File: Core/Security/X509CRL.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+import stat
+import os
+import tempfile
+
+from GSI import crypto
+from DIRAC import S_OK, S_ERROR
+
+class X509CRL:
```
Comment (fstagni) [inline Core/Security/X509CRL.py]: extend from object

**Comment by fstagni** [comment]
Why v6r13? Do you need it for the WebPortal? CC @acasajus for logic checking.

**Comment by zmathe** [comment]
I need for the web portal. It is new and I do not want to wait for v6r14. 
This class is used by the web framework to generate the CRL list. The CRLs list is generated similar way than the CAs.

**Comment by zmathe** [comment]
It is used: https://github.com/DIRACGrid/WebAppDIRAC/pull/198

**Pull request merged**